cmd/ocm/delete/cmd.go

```diff
@@ -40,14 +40,49 @@ var args struct {
 }
 
 var Cmd = &cobra.Command{
-	Use:       "delete [flags] PATH",
+	Use:       "delete [flags] PATH|RESOURCE_ALIAS(with resource_id)",
 	Short:     "Send a DELETE request",
 	Long:      "Send a DELETE request to the given path.",
 	RunE:      run,
 	ValidArgs: urls.Resources(),
 }
 
+var argumentsUsage = map[string]string{
```

Comment:
Noticed that when I rerun the help it generated always in a different order which is caused by the map. Either we could implement some ordered map or having just slice of keys in which the map should print the values which don't feel like would be worth for our case or just use the slice with ordered docs.

Reply:
Will make it consistent